apis/swagger.yml

```diff
@@ -584,6 +584,24 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"   
 
+  /task/metrics:
+    post:
+      summary: "registry a task"
+      description: |
+        Create a peer-to-peer downloading task in supernode.

```

Comment:
To see the endpoint， actually I cannot get the connection of the summary part and description part.

Reply:
Sorry, I will recheck and update it.